src/index.js

```diff
@@ -1307,7 +1314,7 @@ const initializeFormElements = () => {
       params: [
         {
           from: accounts[0],
-          to: '0x4fabb145d64652a948d72533023f6e7a623c7c53',
+          to: `${ERC20_SAMPLE_CONTRACTS[networkName]}`,
```

Comment:
we change dynamically the contract address depending on the network we are in, in order for the transaction to be flagged as malicious, from the sample data Blockaid provided